contracts/registries/IPAssetRegistry.sol

```diff
@@ -22,19 +21,15 @@ import { IPAccountStorageOps } from "../lib/IPAccountStorageOps.sol";
 ///         attribution and an IP account for protocol authorization.
 ///         IMPORTANT: The IP account address, besides being used for protocol
 ///                    auth, is also the canonical IP identifier for the IP NFT.
-contract IPAssetRegistry is IIPAssetRegistry, IPAccountRegistry, Governable {
+contract IPAssetRegistry is IIPAssetRegistry, IPAccountRegistry {
```

Comment:
What's the governance plan for IPAccountRegistry?

Reply:
As the register IP through IPAssetRegistry is permissionlessly. seems no function requires governance. 
However, Pausibility might still be needed.